### PR TITLE
Fixed and improved example code

### PR DESCRIPTION
**Pull request by vanderlokken**

### CODE REVIEW

File: examples/dimension_expander.rs [hunk 1]
```diff
@@ -106,6 +106,7 @@ impl Plugin for DimensionExpander {
             outputs: 2,
             parameters: 2,
             category: Category::Effect,
+            f64_precision: false,
 
             ..Default::default()
```
Comment (zyvitski) [inline examples/dimension_expander.rs]: Rather than setting this flag in each example we should set the default to be false.

**Comment by piedoom** [comment]
There are still some issues with the examples but this PR fixes what it set out to fix, so I submit that we merge this and open up separate issues as we go along.  I'll wait for confirmation from someone else at @rust-dsp

**Comment by zyvitski** [comment]
Just fixed the merge conflicts

**Comment by zyvitski** [comment]
We will need to update the version number in the README for this as well.

**Comment by zyvitski** [comment]
Appveyor failed, will re run ASAP

**Comment by piedoom** [comment]
Good stuff!  Let's merge this one.

**Pull request merged**